db/migrate/20230223075702_add_column_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :text, :string
```

Comment:
- [ ] Same here the `text` should be a `text`, not a `string`, kindly make sure to change the text type to `text` so your migrations reflect the `ERD`